src/player.cpp

```diff
@@ -2977,47 +3042,67 @@ Strength - 4;    Dexterity - 4;    Intelligence - 4;    Perception - 4"));
 
             werase(w_info);
             std::string s;
-            if (line == 0) {
+            if (line == 0) { //Torso
                 const int melee_roll_pen = std::max( -encumb( bp_torso ), -80 );
                 s += string_format( _("Melee attack rolls %+d%%; "), melee_roll_pen );
                 s += dodge_skill_text( - (encumb( bp_torso ) / 10));
                 s += swim_cost_text( (encumb( bp_torso ) / 10) * ( 80 - get_skill_level( skill_swimming ) * 3 ) );
                 s += melee_cost_text( encumb( bp_torso ) );
-            } else if (line == 1) { //Torso
+                s += thrown_cost_text( 2 * encumb( bp_torso ) );
+                s += fall_damage_penalty_text( calc_old_fall_damage_for_encumbrance_penalty( this, bp_torso ) * 100,
+                                               player::fall_damage_mod() * 100 );
+            } else if (line == 1) { //Head
                 s += _("Head encumbrance has no effect; it simply limits how much you can put on.");
-            } else if (line == 2) { //Head
-                s += string_format(_("Perception %+d when checking traps or firing ranged weapons;\n"
-                                     "Perception %+.1f when throwing items."),
-                                   -(encumb(bp_eyes) / 10),
-                                   double(-(encumb(bp_eyes) / 10)) / 2);
-            } else if (line == 3) { //Eyes
-                s += _("Covering your mouth will make it more difficult to breathe and catch your breath.");
+            } else if (line == 2) { //Eyes
+                s += string_format( _( "Perception %+d when checking traps.\n" ), -encumb(bp_eyes) / 10 );
+                s += ranged_penalty_text( -6 * encumb( bp_eyes ) );
+                s += "\n";
```

Comment:
Can be replaced with a char `'\n'` (and better moved into the function). Here and below.